src/test/java/dev/group4/services/UserServiceTests.java

```diff
@@ -0,0 +1,34 @@
+package dev.group4.services;
+
+import dev.group4.entities.User;
+import dev.group4.repos.UserRepo;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class UserServiceTests {
+    private User testUser;
+    @Autowired
+    private UserRepo userRepo;
+    @Order(1)
+    @Test
+    void registerUser() {
+        User user = new User("", "username", "password");
```

Comment:
User user = new User("username", "password");
 
 if it doesn't complain sometimes you need to reset lombock after you change the entity fields because it doesn't change the internal state.